redbox-core/redbox/storage/elasticsearch.py

```diff
@@ -122,22 +93,20 @@ def delete_items(self, items: list[PersistableModel]) -> ObjectApiResponse | Non
             body={"query": {"terms": {"_id": [str(item.uuid) for item in items]}}},
         )
 
+    def delete_user_items(self, model_type: str, user_uuid: UUID, filters: list[dict] = None) -> ObjectApiResponse | None:
+        target_index = f"{self.root_index}-{model_type.lower()}"
+        return self.es_client.delete_by_query(
+            index=target_index,
+            body=ElasticsearchStorageHandler.get_query_match_all_for_user(user_uuid, filters),
+        )
+
     def read_all_items(self, model_type: str, user_uuid: UUID) -> list[PersistableModel]:
```

Comment:
maybe for another PR: is there a `langchain_elasticsearch.ElasticsearchStore` method that does this?